test/integration/garbagecollector/garbage_collector_test.go

```diff
@@ -443,7 +443,7 @@ func setupRCsPods(t *testing.T, gc *garbagecollector.GarbageCollector, clientSet
 	rc.ObjectMeta.Finalizers = initialFinalizers
 	rc, err := rcClient.Create(context.TODO(), rc, metav1.CreateOptions{})
 	if err != nil {
-		t.Fatalf("Failed to create replication controller: %v", err)
+		t.Errorf("Failed to create replication controller: %v", err)
```

Comment:
It seems like `TestStressingCascadingDeletion` should have some way of receiving this err and doing a `Fatalf` instead; maybe use an errs channel as done in ttl_controller_test.go below?

Reply:
Will do!